### PR TITLE
More docs

### PR DESCRIPTION
**Pull request by flovilmart**
after
----
capture d ecran le 2018-08-10 a 14 34 23

### CODE REVIEW

File: src/Adapters/AdapterLoader.js [hunk 1]
```diff
@@ -1,4 +1,15 @@
-export function loadAdapter(adapter, defaultAdapter, options) {
+/**
+ * @module AdapterLoader
+ */
+/**
+ * @static
+ * attempts to load an adapter, or fallbacks to the default adapter
```
Comment (acinader) [inline src/Adapters/AdapterLoader.js]: remove ,

Attempt to load an adapter or fallback to the default.

File: src/Adapters/Analytics/AnalyticsAdapter.js [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
   appOpened(parameters, req) {
     return Promise.resolve({});
   }
 
-  /*
-  @param eventName: the name of the custom eventName
-  @param parameters: the analytics request body, analytics info will be in the dimensions property
-  @param req: the original http request
+  /**
+  @param {String} 8eventName: the name of the custom eventName
```
Comment (acinader) [inline src/Adapters/Analytics/AnalyticsAdapter.js]: typo 8eventName

File: src/Options/Definitions.js [hunk 3]
```diff
@@ -398,3 +398,4 @@ module.exports.LiveQueryServerOptions = {
     "action": parsers.moduleOrObjectParser
   }
 };
+//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IiIsImZpbGUiOiJpbmRleC5qcyIsInNvdXJjZXNDb250ZW50IjpbXX0=
```
Comment (acinader) [inline src/Options/Definitions.js]: did you mean to include source maps here?

File: src/Options/docs.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+ * @property {Adapter<FilesAdapter>} filesAdapter Adapter module for the files sub-system
+ * @property {Any} push Configuration for push, as stringified JSON. See http://docs.parseplatform.org/parse-server/guide/#push-notifications
+ * @property {Boolean} scheduledPush Configuration for push scheduling. Defaults to false.
+ * @property {Adapter<LoggerAdapter>} loggerAdapter Adapter module for the logging sub-system
+ * @property {Boolean} jsonLogs Log as structured JSON objects
+ * @property {String} logsFolder Folder for the logs (defaults to './logs'); set to null to disable file based logging
+ * @property {Boolean} verbose Set the logging to verbose
+ * @property {String} logLevel Sets the level for logs
+ * @property {Boolean} silent Disables console output
+ * @property {String} databaseURI The full URI to your mongodb database
```
Comment (acinader) [inline src/Options/docs.js]: if using postgress, do is this used?

File: src/Options/docs.js [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+ * @property {String} restAPIKey Key for REST calls
+ * @property {String} readOnlyMasterKey Read-only key, which has the same capabilities as MasterKey without writes
+ * @property {String} webhookKey Key sent with outgoing webhook calls
+ * @property {String} fileKey Key for your files
+ * @property {Boolean} preserveFileName Enable (or disable) the addition of a unique hash to the file names
+ * @property {String[]} userSensitiveFields Personally identifiable information fields in the user table the should be removed for non-authorized users.
+ * @property {Boolean} enableAnonymousUsers Enable (or disable) anon users, defaults to true
+ * @property {Boolean} allowClientClassCreation Enable (or disable) client class creation, defaults to true
+ * @property {Any} auth Configuration for your authentication providers, as stringified JSON. See http://docs.parseplatform.org/parse-server/guide/#oauth-and-3rd-party-authentication
+ * @property {String} maxUploadSize Max file size for uploads. defaults to 20mb
```
Comment (acinader) [inline src/Options/docs.js]: uppercase D in Defaults, or use a comma which is done in other cases here

File: src/Options/docs.js [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+ * @property {Boolean} verifyUserEmails Enable (or disable) user email validation, defaults to false
+ * @property {Boolean} preventLoginWithUnverifiedEmail Prevent user from login if email is not verified and PARSE_SERVER_VERIFY_USER_EMAILS is true, defaults to false
+ * @property {Number} emailVerifyTokenValidityDuration Email verification token validity duration
```
Comment (acinader) [inline src/Options/docs.js]: in milliseconds or whatever the denomination is...

File: src/Options/docs.js [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+ * @property {Any} accountLockout account lockout policy for failed login attempts
+ * @property {Any} passwordPolicy Password policy for enforcing password related rules
+ * @property {Adapter<CacheAdapter>} cacheAdapter Adapter module for the cache
+ * @property {Adapter<MailAdapter>} emailAdapter Adapter module for the email sending
```
Comment (acinader) [inline src/Options/docs.js]: remove the 'the'

Adapter module for email sending.

File: src/ParseServer.js [hunk 8]
```diff
@@ -128,6 +131,10 @@ class ParseServer {
     }
   }
 
+  /**
+   * @static
+   * app creates an express app for the parse server
```
Comment (acinader) [inline src/ParseServer.js]: Create an express app for the parse server

File: src/ParseServer.js [hunk 9]
```diff
@@ -239,11 +252,24 @@ class ParseServer {
     return this;
   }
 
+  /**
+   * Creates a new ParseServer adn starts it.
```
Comment (acinader) [inline src/ParseServer.js]: typo 'adn' should be and.

File: src/ParseServer.js [hunk 10]
```diff
@@ -239,11 +252,24 @@ class ParseServer {
...
+   * @param {ParseServerOptions} options used to start the server
+   * @param {Function} callback called when the server started
```
Comment (acinader) [inline src/ParseServer.js]: called when the server has started

missing the 'has'.

**Comment by acinader** [comment]
fyi, i took a look at minimal shortly after we changed to it to see if i could configure it better.  For me, I find very difficult to navigate because the index on the left is too large.  so i usually ctrl - f and search for like object or something.  


just my $.02.  I've goolged around for alternative themes a few times and clearly haven't opened a pr, so again, just me complaining.

**Comment by flovilmart** [comment]
@acinader the goal of having a forked theme is to be able to make it our own, with the things that are painful solved :)

File: src/Options/Definitions.js [hunk 11]
```diff
@@ -84,7 +84,7 @@ module.exports.ParseServerOptions = {
   },
   "databaseURI": {
     "env": "PARSE_SERVER_DATABASE_URI",
-    "help": "The full URI to your mongodb database",
+    "help": "The full URI to your database. Supported databases are mongodb or postres.",
```
Comment (acinader) [inline src/Options/Definitions.js]: postgress is misspelled.

File: src/Options/docs.js [hunk 12]
```diff
@@ -3,19 +3,19 @@
...
- * @property {Boolean} scheduledPush Configuration for push scheduling. Defaults to false.
+ * @property {Boolean} scheduledPush Configuration for push scheduling, defaults to false.
  * @property {Adapter<LoggerAdapter>} loggerAdapter Adapter module for the logging sub-system
  * @property {Boolean} jsonLogs Log as structured JSON objects
  * @property {String} logsFolder Folder for the logs (defaults to './logs'); set to null to disable file based logging
  * @property {Boolean} verbose Set the logging to verbose
  * @property {String} logLevel Sets the level for logs
  * @property {Boolean} silent Disables console output
- * @property {String} databaseURI The full URI to your mongodb database
+ * @property {String} databaseURI The full URI to your database. Supported databases are mongodb or postres.
```
Comment (acinader) [inline src/Options/docs.js]: postgress

File: src/Options/index.js [hunk 13]
```diff
@@ -47,7 +47,7 @@ export interface ParseServerOptions {
   /* Disables console output
   :ENV: SILENT */
   silent: ?boolean;
-  /* The full URI to your mongodb database */
+  /* The full URI to your database. Supported databases are mongodb or postres. */
```
Comment (acinader) [inline src/Options/index.js]: postgress

Comment (flovilmart) [reply src/Options/index.js]: The code is auto generated, so that make sens :)

**Pull request merged**